### DISCUSSION TITLE
[Bug] Name input falls off screen

### DISCUSSION THREAD

**Discussion by TroyRobinson** [webstudio-is/webstudio-community#59] [Q&A]
In the element settings, the name input extends too far, off the screen (see screenshot)

![image](https://github.com/webstudio-is/webstudio/assets/24643126/f041aa4b-5bc9-4b20-841d-c552d9843076)

**Comment by kof** [comment]
I also see width and height to be too long, what's your browser version and OS?

**Reply by kof** [reply]
cc @istarkov

**Comment by kof** [comment]
I believe this was fixed
